src/components/AppStateProvider/AppStateProvider.tsx

```diff
@@ -357,6 +370,12 @@ export const AppStateProvider: React.FC = ({ children }) => {
 
   useEffect(() => {
     navigator.mediaDevices.enumerateDevices().then((devices) => dispatch({ type: 'set-devices', devices }));
+
+    axios('app/token').catch((error) => {
+      if (error.response.data.error.message === 'token server expired') {
+        dispatch({ type: 'set-app-is-expired' });
+      }
+    });
```

Comment:
Okay thanks, that makes sense! Do you think it is worth it to display/handle other potential errors in the app (like the 404 or something else), or not really since the point of this request is just to see if the token has expired?